frontend/src/stores/assignment.ts

```diff
@@ -83,6 +86,59 @@ export const useAssignmentStore = defineStore('assignment', () => {
     });
   }
 
+  function getAssignmentDataAPI(): IAssignmentAPI {
+    const formattedContacts: IAssignmentInfoDataAPI[] = [];
+
+    assignment.contacts.forEach(contact => {
+      const formattedPhoneNumbers: IAssignmentInfoDataAPI[] = [];
+      const formattedAddresses: IAssignmentInfoDataAPI[] = [];
+
+      contact.phoneNumbers.forEach(phoneNumber => {
+        formattedPhoneNumbers.push({
+          type: phoneNumber.type,
+          number: phoneNumber.number
+        })
+      })
+
+      contact.addresses.forEach(address => {
+        formattedAddresses.push({
+          type: address.type,
+          city: address.city,
+          state: address.city,
+          zip: address.zip,
+          addressLine: address.addressLine
+        })
+      })
+
+      formattedContacts.push({
+        type: contact.type,
+        firstName: contact.firstName,
+        lastName: contact.lastName,
+        email: contact.email,
+        phoneNumbers: formattedPhoneNumbers,
+        addresses: formattedAddresses
+      });
+    })
+
+    return {
+      dateOfIncident: format(new Date(assignment.incidentDate), 'yyyy-MM-dd'),
+      contactsInfo: formattedContacts,
+      vehicleInfo: {
+        vinNumber: assignment.vehicleInfo.vinNumber,
+        carMake: assignment.vehicleInfo.carMake,
+        carModel: assignment.vehicleInfo.carModel,
+        yearOfManufacture: +format(new Date(assignment.vehicleInfo.yearOfManufacture), 'yyyy'),

```

Comment:
better to use Number() or parseInt rather than + sign.
also, consider using date-fns to make date conversion clearer

Reply:
Done. There was a mistake: removed a format func